R/vad.R

```diff
@@ -0,0 +1,183 @@
+globalVariables(c("DBZH","VRADH","RHOHV"))
+NULL
+
+#' Create a Velocity Azimuth Display (VAD) plot
+#'
+#' A velocity azimuth display plot visualized the radial velocity (`VRAD`) as a function of the azimuth from the radar.
+#' Among others it can be used to asses the fit of the movement speeds in a vertical profile.
+#' For example, when the movement is not uniform but rather in multiple directions or rotating this will be visible as
+#' deviations from the sine function.
+#'
+#' @param x A polar volume from which range gates are extracted.
+#' @param ... Currently not used.
+#' @param vp A vertical profile to annotate the VAD plot with the fit in the vertical profile
+#' @param range The distance range in to filter the range gates with.
+#'      If a `vp` is provided the range is taken from there and the argument `range` should not be provided.
+#' @param height The height range to filter the range gates by.
```

Comment:
other functions use `alt_min` and `alt_max` to specify height ranges, suggest using the same nomenclature